Countly.m

```diff
@@ -1264,4 +1271,26 @@ - (void)appLoadingFinished
     [CountlyPerformanceMonitoring.sharedInstance recordAppStartDurationTraceWithStartTime:appLoadStartTime endTime:appLoadEndTime];
 }
 
+- (void)halt
+{
+    CLY_LOG_I(@"%s", __FUNCTION__);
+    [self halt:false];
+}
+
+- (void)halt:(BOOL) clearStorage
+{
+    CLY_LOG_I(@"%s %d", __FUNCTION__, clearStorage);
```

Comment:
You did add resettable interface but its logic is not used. The class instances are accessed via themselves not interface. So it is not doing any encapsulation